Libraries/Components/Touchable/TouchableWithoutFeedback.js

```diff
@@ -105,8 +105,8 @@ const PASSTHROUGH_PROPS = [
   'onAccessibilityAction',
   'onBlur',
   'onFocus',
-  'onKeyDown',
-  'onKeyUp',
+  // 'onKeyDown', // [macOS]
+  // 'onKeyUp', // [macOS]
```

Comment:
I figured it was easier to merge if I commented out instead of removing.

Reply:
I think removing is better. This looks like a diff that didn't have a tag to begin with (notice the rest of the passthrough props are below inside a diff block).